src/inngest/functions/setPrimaryCryptoAddressOfUser/logic.ts

```diff
@@ -13,11 +11,12 @@ const zodPrimaryCryptoAddressOfUserParameters = object({
 
 export async function setPrimaryCryptoAddressOfUser(
   parameters: z.infer<typeof zodPrimaryCryptoAddressOfUserParameters>,
+  logger?: Logger,
```

Comment:
Same here, if the idea is make `setPrimaryCryptoAddressOfUser` resilient we should just default this value to the return of `getLogger`
```suggestion
  logger: Logger = getLogger('setPrimaryCryptoAddressOfUser'),
```